package.json

```diff
@@ -55,6 +55,7 @@
     "prettier-eslint": "~9.0",
     "require-directory": "~2.1",
     "require-from-string": "~2.0",
+    "search-insights": "^1.7.1",
```

Comment:
❓ I don't understand why this is needed as the lib is loaded from jsdelivr

Reply:
You probably right, I followed the doc and added the npm dependency first, it didn't work so I loaded the library from jsdelivr as suggested and it worked. I should have tried to dig a bit further to understand what was the issue with the npm dependency.